rental.rb

```diff
@@ -0,0 +1,24 @@
+require 'date'
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @book = book
+    @book.rentals << self
+    @person = person
+    @person.rentals << self
+    @date = date
+  end
+
+  def book=(book)
+    @book = book
+    @book.rentals.push(book) unless @book.rentals.member?(book)
+  end
+
+  def person=(person)
+    @person = person
+    @person.rentals.push(person) unless @person.rentals.member?(person)
+  end
+end
```

Comment:
- [optional] I wouldn't know your intentions for these two methods, but I suggest you remove them since the constructor had already catered for book and person belongs-to association. ℹ️